code-challenges.js

```diff
@@ -35,6 +69,27 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+// declare function noVowel that takes an argument (string)
+const noVowel = (string) => {
+  // declare a new let array #vowels that identifies the array "a", "e", "i", "o", "u", "A", "E", "I", "O", "U"
+    let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
+    // create a new local array  "woVowels" that
+    let splitString = string.split("")
+    // fliter through (array) against #vowels
+
+    let filtered = splitString.filter(letter =>
+          // learned about .includes
+       !vowels.includes(letter))
+      console.log(splitString);
+      console.log(filtered);
+
+            // return woVowels.join as an string
+        return filtered.join("")
+}
+// call function
+console.log(noVowel(stringWithVowels1));
+console.log(noVowel(stringWithVowels2));
```

Comment:
great work and great pseudocode! again, you can just return the filtered array if you'd like :)